src/features/PatientInfo/index.js

```diff
@@ -16,39 +16,10 @@ function PatientInfo(props) {
       <h1 className="patient-name">{input.name}</h1>
       <div className="patient-detail">
         <div className="id">
-          {/* <div className="float-end"><a href="fake_ehr.html" className="view">View patient in EHR</a></div> */}
+          <div><b>Date of Birth:</b> <span><time dateTime={dob}>{dob}</time></span> ({input.age} y.o.)</div>
           <div><b>MRN:</b> <span>{mrn}</span></div>
           <div><b>Pregnant:</b> <span>{input.isPregnant === false ? 'No' : input.isPregnant === true ? 'Yes' : null}</span></div>
         </div>
-        <div className="info-items">
-          <Row>
-            <Col>
-              <div className="info-item">
-                <b>Date of Birth</b> <span><time dateTime={dob}>{dob}</time></span>
-              </div>
-              <div className="info-item">
-                <b>Sex at Birth</b>
-                <span>{input.sexAtBirth}</span>
-              </div>
-            </Col>
-            <Col>
-              <div className="info-item">
-                <b>Age</b> <span>{input.age}</span>
-              </div>
-              <div className="info-item">
-                <b>Gender</b> <span>{gender}</span>
```

Comment:
Eslint is now throwing a warning on line 12 because gender is no longer used. I would be fine with keeping it, in case we do want to add it back in the future. But in that case I would suggest we disable the eslint rule for line 12, and make a comment about gender being unused.

Reply:
I addressed this in https://github.com/ccsm-cds-tools/ccsm-cds-dashboard/pull/54/commits/16cc449155dc5185518678320c6efb0d8ce5bba1.